### PR TITLE
LPD-39869

### PR DESCRIPTION
**Pull request by gabrielwas**
https://liferay.atlassian.net/browse/LPD-39869

### CODE REVIEW

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

File: modules/util/portal-tools-service-builder-test-service/src/main/java/com/liferay/portal/tools/service/builder/test/object/system/ERCCompanyEntrySystemObjectDefinitionManager.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+		ERCCompanyEntry ercCompanyEntry =
+			_ercCompanyEntryLocalService.createERCCompanyEntry(
+				_counterLocalService.increment());
+
+		ercCompanyEntry.setExternalReferenceCode(
+			GetterUtil.getString(values.get("externalReferenceCode")));
+		ercCompanyEntry.setColumn1(
+			GetterUtil.getInteger(values.get("column1")));
+
+		ERCCompanyEntry persistedERCCompanyEntry =
```
Comment (carolmariaabb) [inline modules/util/portal-tools-service-builder-test-service/src/main/java/com/liferay/portal/tools/service/builder/test/object/system/ERCCompanyEntrySystemObjectDefinitionManager.java:54]: @gabrielwas why did you have to create another variable here? can't we just reuse the existing one?

File: modules/util/portal-tools-service-builder-test-test/src/testIntegration/java/com/liferay/portal/tools/service/builder/test/ERCCompanyEntrySystemObjectDefinitionManagerTest.java
Comment (carolmariaabb) [inline modules/util/portal-tools-service-builder-test-test/src/testIntegration/java/com/liferay/portal/tools/service/builder/test/ERCCompanyEntrySystemObjectDefinitionManagerTest.java:1]: I really liked the test. It makes a lot of sense to me.

**Review by carolmariaabb — APPROVED** [review]

File: modules/util/portal-tools-service-builder-test-service/src/main/java/com/liferay/portal/tools/service/builder/test/object/system/ERCCompanyEntrySystemObjectDefinitionManager.java (hunk 1 above)
Comment (gabrielwas) [reply modules/util/portal-tools-service-builder-test-service/src/main/java/com/liferay/portal/tools/service/builder/test/object/system/ERCCompanyEntrySystemObjectDefinitionManager.java]: @carolmariaabb good! Thx.

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
Sending to Brian, since it was just SF changes

**Comment by gabrielwas** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/155879

**Pull request closed without merging**